src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -473,18 +473,18 @@ public WorkspacePanel(final MCreator mcreator) {
 
 		se.add("West", leftPan);
 
-		JScrollablePopupMenu filterPopup = new JScrollablePopupMenu();
+                JScrollablePopupMenu filterPopup = new JScrollablePopupMenu();

```

Comment:
Extra indentation is not needed.

Reply:
? i didn't touch this

Reply:
![image](https://user-images.githubusercontent.com/60476871/118640790-72bab100-b7ea-11eb-99e4-285ab3ddda63.png)

It's where it should be in my Fork

Reply:
weird thing